product_docs/docs/pgd/5/manual/08-using-pgd-cli.mdx

```diff
@@ -0,0 +1,252 @@
+---
+title: Step 8 - Using PGD CLI
+navTitle: Using PGD CLI
+deepToC: true
+---
+
+## Using PGD CLI
```

Comment:
General comment on this section is that it should probably just refer to the existing PGD CLI installation docs section, vs. duplicating any of that content.  Having the worked example is still valuable, but I don't think we should duplicate anything here that is already covered here https://deploy-preview-5104--edb-docs-staging.netlify.app/docs/pgd/latest/cli/, and this section should have a link to the full-coverage in the other section.

Reply:
You could say the same of PGD Proxy docs though. This is intended to get the user fully equipped.

Though really, there should be a part 7/8 for configuring psql and .pgpass for connecting from a client. That I will leave for another day.

I have trimmed this section slightly and reduced repetition.